compat/src/main/scala-2.11/scala/collection/compat/package.scala

```diff
@@ -0,0 +1,48 @@
+package scala.collection
+
+import scala.collection.generic._
+import scala.collection.{immutable => i, mutable => m}
+
+package object compat extends compat.PackageShared {
+  implicit def toImmutableSortedMapExtensions(fact: i.SortedMap.type): compat.ImmutableSortedMapExtensions =
+    new compat.ImmutableSortedMapExtensions(fact)
```

Comment:
A previous PR was merged but was broken for 2.11. This fix this.

Reply:
This PR disable the CI: https://github.com/scala/scala-collection-compat/pull/109
This PR does not compile in Scala 2.11: https://github.com/scala/scala-collection-compat/pull/102

I'm splitting the package object into two parts: 2.11 and 2.12. I have to do this because mutable.TreeMap is only available since 2.11. The package object makes it hard to reuse code between 2.11 and 2.12.